concent_api/common/decorators.py

```diff
@@ -47,3 +51,15 @@ def wrapper(*args: Any, **kwargs: Any) -> None:
                 logging_level=LoggingLevel.ERROR)
             raise
     return wrapper
+
+
+def non_nesting_atomic(using: str=None, savepoint: bool=True) -> Atomic:
```

Comment:
```suggestion
def non_nesting_atomic(using: str, savepoint: bool=True) -> Atomic:
```
it seems that all calls to the decorator contain a value for `using`. In other words it is never called with `None`, so both the signature and `get_connection()` call should be changed.

Reply:
It is, but it is also not required to work properly, so if someone wants to use it without it, he can. Please check `transaction.atomic` code for reference.

Reply:
:+1: done